build-system/common/esbuild-babel.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+const argv = require('minimist')(process.argv.slice(2));
```

Comment:
Nit: no need to process it, just get the raw array.
```suggestion
const argv = process.argv.slice(2);
```

Reply:
Or remove this `require` and directly use `process.argv` in `optionsHash`?

Reply:
both ideas work